tests/conftest.py

```diff
@@ -61,12 +248,18 @@ def test_adapter(request):
         pytest.skip("{} cannot be instantiated".format(str(request.cls.tadapter)))
 
 
+def assert_core(expected, time, test_core, name, region, unit, time_points):
+    pview = test_core.parameters.get_timeseries_view(
+        name, region, unit, time_points, ParameterType.POINT_TIMESERIES
+    )
+    relevant_idx = (np.abs(time_points - time)).argmin()
+    np.testing.assert_allclose(pview.get()[relevant_idx], expected)
+
+
 @pytest.fixture(scope="function")
 def test_run_parameters():
-    run_parameters = namedtuple("RunParameters", ["start_time", "stop_time"])
```

Comment:
@znicholls namedtuple is a factory which returns a class. That class should then be used to instantiate objects

Reply:
nice thanks!